app/lib/routes/inner_navigator/new_travel_route.dart

```diff
@@ -14,6 +15,8 @@ class NewTravelRouteGenerator {
             return TestScreenOne();
           case TestScreenTwo.route:
             return TestScreenTwo();
+          case UploadImage.route:
```

Comment:
@MummenRider is it a good idea to dedicate a screen for images/camera? Would we want it to be an overlay without bottom nav?

Reply:
It would look weird  if there's a bottom nav on the camera page. For now, we can use what we have here since this is just experimentation.